custom_components/plugwise/binary_sensor.py

```diff
@@ -12,15 +12,25 @@
     BinarySensorEntityDescription,
 )
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import EntityCategory
+from homeassistant.const import ATTR_NAME, EntityCategory
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .const import (
+    BINARY_SENSORS,
+    COMPRESSOR_STATE,
+    COOLING_ENABLED,
+    COOLING_STATE,
     COORDINATOR,  # pw-beta
+    DHW_STATE,
     DOMAIN,
+    FLAME_STATE,
+    HEATING_STATE,
     LOGGER,
+    NOTIFICATIONS,
+    PLUGWISE_NOTIFICATION,
```

Comment:
Why not just 'notification'?

Reply:
In principle OK but I would like to keep the naming as close to the original string.
Also it's a single present string. Different from all the ELECTRICITY.... sensors.